layer/exceptions/exceptions.py

```diff
@@ -273,7 +275,7 @@ def __init__(self, message: str):
 
 
 class LayerServiceUnavailableExceptionDuringExecution(ProjectExecutionException):
-    def __init__(self, run_id: RunId, message: str):
+    def __init__(self, run_id: uuid.UUID, message: str):
```

Comment:
if RunId is an important concept, we could introduce one in `contracts` then if using `uuid.UUID` instead reduces clarity and understandability of the code?

Reply:
yes, we can do that in the next iteration of adding domain objects